src/tokenize.js

```diff
@@ -548,13 +548,38 @@ pp.readString = function(quote) {
 
 // Reads template string tokens.
 
+const INVALID_TEMPLATE_ESCAPE_ERROR = {}
+
+pp.tryReadTemplateToken = function() {
```

Comment:
Should this method return something? It currently calls `readTmlToken` and (sometimes) `readInvalidTemplateToken`, but discards the results from both calls.